test/e2e/service.go

```diff
@@ -47,129 +46,6 @@ var _ = Describe("Services", func() {
 		namespace1 = "e2e-ns-" + dateStamp() + "-1"
 	})
 
-	It("should provide DNS for the cluster", func() {
```

Comment:
Yes, DNS is a multi-port service, so that's giving us coverage of the
feature.  Creating our own service with 2 ports (which is what you were
doing, right?) and testing it all the way through would be a good
subsititute.

On Thu, May 14, 2015 at 8:31 AM, marekbiskup <notifications@github.com>
wrote:

> In test/e2e/service.go
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/8230#discussion_r30330797>
> :
>
> > @@ -47,129 +46,6 @@ var _ = Describe("Services", func() {
> >  		namespace1 = "e2e-ns-" + dateStamp() + "-1"
> >  	})
> >
> > -	It("should provide DNS for the cluster", func() {
>
> @thockin <https://github.com/thockin>
> I must say I'm a bit lost. Is DNS the multiport service that is tested
> here? (tested by dig)
>
> Shall I test a custom service (like a web server created in a pod, with
> multiple ports)?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/8230/files#r30330797>
> .
>